README.adoc

```diff
@@ -69,16 +64,15 @@ mvn clean install liberty:start-server
 ```
 
 Point your browser to the `http://localhost:9080/inventory/systems/localhost` URL, which accesses the
-`inventory` service with a localhost host name. You see the system properties for this host.
-When you visit this URL, some of these system
-properties, such as the OS name and user name, are automatically stored in the inventory.
+`inventory` service with a localhost host name. You see all of the system properties for this host from the `system` service.
```

Comment:
I think that we should format localhost as a code phrase, `localhost`.

Reply:
We should probably format it like this across the whole doc.

Reply:
There are 2 occurrences of `localhost` being referred to as host name. Those are fixed with back ticks. Thanks.